src/main/java/com/casper/sdk/service/CasperService.java

```diff
@@ -235,21 +235,11 @@ DictionaryData getStateDictionaryItem(@JsonRpcParam("state_root_hash") String st
     @JsonRpcMethod("info_get_status")
     StatusData getStatus();
 
-
     /**
      * Returns an AddressableEntity from the network
      *
      * @param entityIdentifier entity identifier
-     * @return object holding the AddressableEntity or LegacyAccount
-     */
-    @JsonRpcMethod("state_get_entity")
-    StateEntityResult getStateEntity(@JsonRpcParam("entity_identifier") EntityIdentifier entityIdentifier);
-
-    /**
-     * Returns an AddressableEntity from the network
-     *
-     * @param entityIdentifier entity identifier
-     * @param blockIdentifier block identifier
+     * @param blockIdentifier  block identifier
```

Comment:
Please document why block identifier is optional. Latest block is used if not provided